src/it/projects/pom-create-lock-file-with-integrity-checking/expected-pom.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd"
+  xmlns:integrity="urn:se.vandmo.dependencylock.integrity">
```

Comment:
I haven't seen urn:... used before but it kind of makes more sense then pointing to a http location that doesn't exist :+1: 
Should probably be just `urn:se.vandmo.dependencylock` though.

Reply:
Changed to that.